test-containers/bnd-tools-smoke-test/app/app.bndrun

```diff
@@ -36,5 +36,5 @@
 	bnd.identity;version='${flow.version}';id='com.vaadin.flow.html.components',\
 	bnd.identity;version='${flow.version}';id='com.vaadin.flow.push',\
 	bnd.identity;version='${flow.version}';id='com.vaadin.flow.osgi',\
-	bnd.identity;version='1.1.1';id='com.vaadin.license-checker'
+	bnd.identity;version='1.10.0';id='com.vaadin.license-checker'
```

Comment:
Shouldn't license checker version be 1.11.2 as in POM file?

Reply:
Yes, good catch